runtime/reference/cli/install.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "`deno install`"
+oldUrl:
+ - /runtime/manual/tools/script_installer/
+ - /runtime/reference/cli/script_installer/
+ - /runtime/manual/tools/script_installer/
+ - /runtime/manual/tools/cache/
+ - /runtime/reference/cli/cache/
+---
+
+`deno install` is a tool that allows you to install remote dependencies to use
+in your project or as executables available on your machine.
+
+There are four ways to use `deno install`:
+
+- `deno install` - install all dependencies specified in `deno.json` and
+  `package.json`
+- `deno install [PACKAGES]` - install and add specified dependencies to
+  `deno.json` or `package.json`
+- `deno install --entrypoint [FILES]` - install all remote dependencies
+  discovered from the provided files
+- `deno install --global [PACKAGE_OR_URL]` - install a dependency as an
+  executable program on your machine
+
+:::info
+
+`deno install` works in similar manner to other package managers like `npm`,
+`yarn`, `pnpm` or `bun`.
+
+:::
+
+## `deno install`
```

Comment:
I've been mostly not using back ticks in headings because they don't look uniform or read as clearly (imho)

Reply:
Removed backticks